backend/whisper_with_diarization_as_methods.py

```diff
@@ -283,14 +283,29 @@ def write_list_to_csv(list_of_csv_content, output_csv_path: str, output_csv_head
     :param list_of_csv_content:
     :return:None
     """
-    with open(output_csv_path, "w") as comb_lang_csv_file:
+    with open(output_csv_path, "w", encoding = "utf-8") as comb_lang_csv_file:
         comb_lang_csv_writer = csv.writer(comb_lang_csv_file)
         comb_lang_csv_writer.writerow(output_csv_headers)  # Write the header row
         for i in range(len(list_of_csv_content)):
             comb_lang_csv_writer.writerow(list_of_csv_content[i])
     comb_lang_csv_file.close()
+    
+def obtain_translation_transcription_content_multiple_speakers(whisper_result, diarization_result) -> list: 
+    """This function returns a list that is formatted for the output CSV file for multiple speakers."""
+    final_result = display_timestamps_speaker_and_text(whisper_result, diarization_result)
+    csv_content = writing_solo_res_to_csv(final_result)
+    return csv_content
+    
+def obtain_translation_transcription_content_single_speaker(whisper_result) -> list: 
```

Comment:
You can probably shorten the method name to something like obtain_output_content_single_speaker